win32_deps_build.sh

```diff
@@ -5,12 +5,6 @@ set -e
 SCRIPT_DIR="$(dirname "$BASH_SOURCE")"
 SCRIPT_DIR="$(realpath "$SCRIPT_DIR")"
 
-USE_MINGW_LLVM=${USE_MINGW_LLVM:-}
```

Comment:
Since you aren't adding explicit checks for variables that are supposed to be set by `win32_build.sh`, I would suggest setting `set -u` at the top to abort on trying to expand an unset variable.